SignalMessaging/profiles/OWSUserProfile.m

```diff
@@ -106,17 +106,11 @@ - (void)setAvatarUrlPath:(nullable NSString *)avatarUrlPath
 {
     @synchronized(self)
     {
-        BOOL isEqual;
-        if (avatarUrlPath != nil) {
-            isEqual = [avatarUrlPath isEqual:_avatarUrlPath];
-        } else {
-            // are they both nil?
-            isEqual = _avatarUrlPath == nil;
-        }
+        BOOL didChange = ![NSObject isNullableObject:_avatarUrlPath equalTo:avatarUrlPath];
```

Comment:
WDYT?  I also use this new method in the backup work.

Reply:
I think this is great. Thanks!